client/state/stats/lists/selectors.js

```diff
@@ -119,3 +117,28 @@ export function getSiteStatsPostsCountByDay( state, siteId, query, date ) {
 	return data[ date ] || null;
 }
 
+/**
+ * Returns a parsed object of statsPublicize data for a given query, or default "empty" object
+ * if no statsStreak data has been received for that site.
+ *
+ * @param  {Object}  state    Global state tree
+ * @param  {Number}  siteId   Site ID
+ * @param  {Object}  query    Stats query object
+ * @return {Array}            Parsed Data for the query
+ */
+export const getSiteStatsParsedData = createSelector(
+	( state, siteId, statType, query ) => {
+		const data = getSiteStatsForQuery( state, siteId, statType, query );
+
+		if ( 'function' === typeof utils[ statType ] ) {
+			return utils[ statType ].call( this, data );
```

Comment:
Maybe instead of adding these as base exports in `utils.js`, you could combine them into an object, e.g. `Parsers` and call `import { getSerializedStatsQuery, Parsers } from './utils';` ?